### PR TITLE
Added output to acknowledge invalid commands

### LINKED ISSUE #7: Prevent invalid input on keyboard inputs
Validate when command is invalid, display a message like:

image

### PR DESCRIPTION
**Pull request by krisstee**
**CHANGELOG** :memo:

*

**PRINTS(OPTIONAL)** 
Screen Shot 2020-10-27 at 10 11 14 PM

**LABELS**
enhancement
* Example: enhancement, wip etc

Fixes: Issue #7

### CODE REVIEW

**Comment by krisstee** [comment]
Thank you for allowing me to contribute! I am very new to working with TypeScript, so please let me know if there are any further changes that could me made.

**Comment by pyymenta** [comment]
Hello @krisstee !! Thank you about your contribution! Let me review it

File: src/keyListener.ts [hunk 1]
```diff
@@ -39,13 +40,14 @@ export default class KeyListener {
...
+                this.validKeys.includes(key.name) &&
                 (Array.isArray(e.keyName) ? e.keyName.includes(key.name) : e.keyName === key.name) &&
                 (e.ctrl ? key.ctrl : true) &&
                 (e.shift ? key.shift : true),
         );
 
         if (foundRelatedEvent) {
             foundRelatedEvent.handler(key.name, key.ctrl, key.shift);
-        }
+        } else console.log('Invalid command.\n');
```
Comment (pyymenta) [inline src/keyListener.ts]: Here you should wrap the log instruction inside blocks, like this:

```suggestion
        } else {
            console.log('Invalid command.\n');
        }
```

**Review by pyymenta — APPROVED** [review]
Besides that, looks good to me! Good job! :rocket: :rocket: :rocket:

File: src/keyListener.ts (hunk 1 above)
Comment (krisstee) [reply src/keyListener.ts]: Agreed! I applied this suggestion to a second commit.

**Comment by pyymenta** [comment]
@krisstee You did it! I'll merge in! Thank you! :tada: :tada: :tada:

**Pull request merged**